src/test/java/com/google/planningworks/cdap/plugins/base/AnaplanPluginConfigTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.planningworks.cdap.plugins.base;
+
+import static com.anaplan.client.AnaplanService.NAME_AUTH_SERVICE_LOCATION;
+import static com.anaplan.client.AnaplanService.NAME_MODEL_ID;
+import static com.anaplan.client.AnaplanService.NAME_PASSWORD;
+import static com.anaplan.client.AnaplanService.NAME_SERVICE_LOCATION;
+import static com.anaplan.client.AnaplanService.NAME_USERNAME;
+import static com.anaplan.client.AnaplanService.NAME_WORKSPACE_ID;
```

Comment:
style: didn't catch this before, but our projects don't import static variables. Instead, import AnaplanService and refer to them in this class as AnaplanService.NAME_MODEL_ID, etc

Reply:
Got, updated!